.eslintrc.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
         devDependencies: true,
       },
     ],
+    "lines-between-class-members": [
```

Comment:
Not sure why this hasn't been picked up by the linter. Can the double quotes become single quotes please? And if the linter was to fail if this happened against all the better.

Reply:
I'll change it to single quotes for consistency, but single quotes in JSON is not the standard format, according to the JSON spec:

A value can be a string in double quotes, or a number, or true or false or null, or an object or an array. These structures can be nested

http://www.json.org/

Reply:
The yarn.lock file for this commit includes the mocha upgrade, I'll close the other PR

Reply:
Regarding the lock file it looks like it is already on `mocha@5+`. The `package.json` will need updating to keep greenkeeper happy.